service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionService.java

```diff
@@ -91,104 +92,114 @@ private DocmosisJudgeAndLegalAdvisor getJudgeAndLegalAdvisor(Order standardDirec
             .build();
     }
 
-    //TODO: look into and potentially refactor -> handled in hearingService?
-    private DocmosisHearingBooking getHearingBookingData(CaseData caseData) {
-        if (caseData.getHearingDetails() == null || caseData.getHearingDetails().isEmpty()) {
-            return DocmosisHearingBooking.builder().build();
-        }
-
-        HearingBooking prioritisedHearingBooking = hearingBookingService.getMostUrgentHearingBooking(caseData
-            .getHearingDetails());
-
-        HearingVenue hearingVenue = hearingVenueLookUpService.getHearingVenue(prioritisedHearingBooking.getVenue());
+    private List<DocmosisChildren> getChildrenDetails(List<Element<Child>> children) {
+        return children.stream()
+            .map(element -> element.getValue().getParty())
+            .map(this::buildChild)
+            .collect(toList());
+    }
 
-        return DocmosisHearingBooking.builder()
-            .hearingDate(dataExtractionService.getHearingDateIfHearingsOnSameDay(prioritisedHearingBooking).orElse(""))
-            .hearingVenue(hearingVenueLookUpService.buildHearingVenue(hearingVenue))
-            .preHearingAttendance(dataExtractionService.extractPrehearingAttendance(prioritisedHearingBooking))
-            .hearingTime(dataExtractionService.getHearingTime(prioritisedHearingBooking))
-            .hearingJudgeTitleAndName(formatJudgeTitleAndName(prioritisedHearingBooking.getJudgeAndLegalAdvisor()))
-            .hearingLegalAdvisorName(getLegalAdvisorName(prioritisedHearingBooking.getJudgeAndLegalAdvisor()))
+    // TODO: default value is used here for gender, is this correct?
```

Comment:
I don't think we should put that kind of comments in the code, it's not the place for asking business questions. But it makes sense to link to Jira ticket as you did on line 56.

Reply:
hmm, so there is JIRA ticket FPLA-1087, which should be getting actioned very soon. It's against policy to have "BLANK - please complete" in our orders. I think I prefer that this is signposted as it should be incredibly easy to see where have default values and where we might need to change etc 🤔

Reply:
but I will replace with JIRA ticket number